tests/koops-parser.at

```diff
@@ -11,19 +11,22 @@ int run_test(const struct test_struct *test)
 {
 	FILE *fp = xfopen_ro(test->filename);
 	fprintf(stderr, "%s\t", test->filename);
-	g_autofree char *line = NULL;
-	g_autofree char *version = NULL;
+	char *line;
+	char *version;
 	while ((line = libreport_xmalloc_fgetline(fp)) != NULL)
 	{
 		version = abrt_koops_extract_version(line);
+        free(line);
```

Comment:
Minor nitpick, the file uses tabs for indentation while your new code uses spaces.